src/views/Yummy/yummy.scss

```diff
@@ -0,0 +1,35 @@
+/* Top section css */
+.top-section {
+  display: flex;
+  justify-content: center; 
+  align-items: center;
+  margin-top: 3%; 
+}
+.plate {
+  width: 50%;
+}
+.fork, .knife {
```

Comment:
kan hier net zo goed een class utensils of iets van maken, inplaats van twee keer dezelfde class die hetzelfde doet